### PR TITLE
Add initial GitHub readme with markdown

### PR DESCRIPTION
**Pull request by un1versal**
Add initial project readme.
All Github projects have one, KVIrc should be no exception. 

To preview see https://github.com/uNiversaI/KVIrc/blob/Mary-had-a-little-lamb/README.md

Please provide feedback on things to add / remove or improve. before merging but things can always be added after

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,39 @@
-[![Travis Build Status](https://travis-ci.org/kvirc/KVIrc.svg?branch=master)](https://travis-ci.org/kvirc/KVIrc)
-[![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/j6tjel0eaeyixcbn/branch/master?svg=true)](https://ci.appveyor.com/project/DarthGandalf/kvirc/branch/master)
+[![Travis Build Status](https://travis-ci.org/kvirc/KVIrc.svg?branch=master)](https://travis-ci.org/kvirc/KVIrc) [![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/j6tjel0eaeyixcbn/branch/master?svg=true)](https://ci.appveyor.com/project/DarthGandalf/kvirc/branch/master)
```
Comment (DarthGandalf) [inline README.md:1]: Why do you put them on one line?

Comment (un1versal) [inline README.md:1]: They were on one line before, looks exactly the same. Github markdown is funny like that.

Comment (DarthGandalf) [inline README.md:1]: They were on separate lines in the source. It renders the same, but IMHO it's cleaner to keep them separate.

Comment (un1versal) [inline README.md:1]: fine

**Pull request merged**